ci/test/pipeline.template.yml

```diff
@@ -486,6 +486,20 @@ steps:
       queue: linux-x86_64
     coverage: skip # Randomized, shouldn't count for coverage
 
+  - id: persist-txn-maelstrom
```

Comment:
You may wish to put this in Nightly first and then promote it to the main CI if there are any concerns that there will be random failures due to the randomization.